tests/v2/validation/pipeline/scripts/setup_registries_environment.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -ex
-cd $(dirname $0)/../../../../
```

Comment:
is there a better way to get the path for these?

Reply:
I'm opened to that too, but at least this way we don't have hardcoded absolute path and it works locally and on Jenkins. Dev does a similar thing in their drone scripts.

Reply:
I think it's more like a personal preference, I am sure we can do something different either in Jenkinsfile or in the script part. In my opinion, since we go to do the work path and build the binaries from there, it makes it easier to see where the bin and the glue code/package are.